doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -220,8 +220,15 @@ def go_to_pose_goal(self):
 
     move_group.set_pose_target(pose_goal)
 
-    ## Now, we call the planner to compute the plan and execute it.
-    plan = move_group.go(wait=True)
+    ## We call the planner to compute the plan and execute it.
```

Comment:
```suggestion
    ## We call the planner to compute the plan.
```

Reply:
the `go` command does execute it though. I would be good with `We call the move_group_interface to compute the plan and execute it.`

Reply:
Right.